### DISCUSSION TITLE
Adjustable transparency of layers

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-website-jekyll#35] [Riverscapes Viewer] [answered]
In the Webinar yesterday, a couple users had brought up issue of being able to adjust the transparency of individual layers. I think @MattReimer that you had started working on this or at least had a ticket somewhere  for it? Can you confirm? This is a useful feature.

Related to this, watching some users, they were reluctant to remove the hillshades, which made it harder for them to take advantage of some of the basemap context. In a round-about way, I think if they adjusted the transparency of the hillshade, that might help them. 

Also related, I noticed the icon size of the zoom to layer extent button is really large and people accidently click that instead of remove layer.

**Comment by MattReimer** [comment, marked as answer]
This was always in the plan for a future version. Being able to drag-reorder layers and being able to change transparency are critical. 

For now I'm tracking this here but I may break it into separate issues to work on it.

- https://github.com/Riverscapes/rs-web-monorepo/issues/219
